frontend/packages/console-dynamic-plugin-sdk/src/extensions/create-operand.ts

```diff
@@ -0,0 +1,27 @@
+import { match as RouterMatch } from 'react-router';
+import { ExtensionK8sModel } from '../api/common-types';
+import { CodeRef, Extension, ExtensionDeclaration } from '../types';
+
+export type CreateOperand = ExtensionDeclaration<
+  'console.olm/create-operand',
```

Comment:
Do we want to make this a general "create resource" extension point that is used for both OLM operands and list pages (for example, when the Create button is clicked on Home -> Search).

Reply:
We should strive for a good balance between the number of extension points vs. their generic-ness (more generic means more properties and thus more complexity). Extension properties wise, if the overlap/intersection between OLM operands and list pages is relatively high, +1 to have just one extension point.

Reply:
I think we can reuse this same extension across all resources for their `/~new` page. 
How would a new operand differ to any other new resource?